sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -676,7 +676,7 @@ class CustomFormModel(object):
         The date and time (UTC) when model training was requested.
     :ivar ~datetime.datetime completed_on:
         Date and time (UTC) when model training completed.
-    :ivar list[~azure.ai.formrecognizer.CustomFormSubModel] models:
+    :ivar list[~azure.ai.formrecognizer.CustomFormSubModel] submodels:
```

Comment:
if we are using `submodel` should we rename to `CustomFormSubmodel`?
Anne pointed that out in .NET PR => https://github.com/Azure/azure-sdk-for-net/pull/12313#discussion_r431233885

Reply:
thanks, that makes sense! let me update the issues